### PR TITLE
Now supports procedure library documentation.

### PR DESCRIPTION
**Pull request by aptmterry**
Now supports response map documentation (optional).

Supports changeDate parameter to prevent regeneration of documentation
prior to specified date.

Supports uriDocumentationList parameter. If the specified URI is
present, only document the projects specified within the text file. This
overrides the changeDate parameter.

Displays warning messages for any missing headline or description.

Added documentation to this tool.

### CODE REVIEW

File: Libraries/Tools/Community/ui_documentation/README.md [hunk 1]
```diff
@@ -1 +1,62 @@
-# project://ui_documentation
+Project: Documentation Generator
+Description: Generate project documentation, for Git repository, from QuickCall and Procedure libraries. 
+Category: automation
```
Comment (cwhitty) [inline Libraries/Tools/Community/ui_documentation/README.md]: this should be library, not automation, correct? it currently exists under Libraries in the rep

**Comment by aptmterry** [comment]
It's a tool to automate documentation. I can change to library.

**Comment by aptmterry** [comment]
This pull request also includes "updated project documentation" for the projects that didn't conform to the new readme.txt template. Also changed ui_documentation to library.

**Comment by cwhitty** [comment]
thanks @aptmterry

**Pull request merged**